client/blocks/comments/post-comment-list.jsx

```diff
@@ -212,35 +212,41 @@ class PostCommentList extends React.Component {
 
 		const {
 			commentsFilter,
+			commentsTree,
+			haveMoreCommentsToFetch,
 			showFilters,
+			totalCommentsCount,
 		} = this.props;
 
 		const {
+			amountOfCommentsToTake
+		} = this.state;
+
+		const {
 			displayedComments,
 			displayedCommentsCount
-		} = this.getDisplayedComments( this.props.commentsTree.get( 'children' ), this.state.amountOfCommentsToTake );
+		} = this.getDisplayedComments( commentsTree.children, amountOfCommentsToTake );
 
 		// Note: we might show fewer comments than totalCommentsCount because some comments might be
 		// orphans (parent deleted/unapproved), that comment will become unreachable but still counted.
-		const showViewEarlier = ( this.props.commentsTree.get( 'children' ).size > this.state.amountOfCommentsToTake ||
-								this.props.haveMoreCommentsToFetch );
+		const showViewEarlier = ( size( commentsTree.children ) > amountOfCommentsToTake || haveMoreCommentsToFetch );
```

Comment:
`size` works with array, string and object. Is `commentsTree.children` an object or array here?